fs/remote/resolver.go

```diff
@@ -299,6 +301,7 @@ type fetcher struct {
 	urlMu         sync.Mutex
 	tr            http.RoundTripper
 	blobURL       string
+	digest        string
```

Comment:
The type of `digest` should be `digest.Digest` rather than string.

Reply:
Done.